LibTrixi.jl/src/api_c.jl

```diff
@@ -328,32 +373,60 @@ trixi_nvariables_cfptr() = @cfunction(trixi_nvariables, Cint, (Cint,))
 
 
 """
-    trixi_load_cell_averages(data::Ptr{Cdouble}, simstate_handle::Cint)::Cvoid
+    trixi_load_cell_averages(data::Ptr{Cdouble}, index::Cint, simstate_handle::Cint)::Cvoid
```

Comment:
As mentioned before, remember to make a breaking release after this change. It might be good to think whether other breaking changes can be included as well before the next release, e.g., regarding the cell/element naming choice.